packages/unmock-core/src/__tests__/dynamicService.test.ts

```diff
@@ -88,5 +88,25 @@ describe("Tests dynamic path tests", () => {
       .nock("https://abc.com", "foo")
       .get("foo")
       .reply(500);
+    // @ts-ignore // "private fields" heh...
+    expect(unmock.services.foo.core.oasSchema.servers).toEqual([
```

Comment:
Maybe services could expose which servers they apply to so one would not need to go this deep into internals.

Reply:
I'd rather keep those things abstracted away from the end-user.
This was just the simplest way to test it in *this* file (the other way would be to put this in a backend test and then just try and access both the URLs).